src/main/java/org/coffeemine/app/spring/UserProfileComponent.java

```diff
@@ -0,0 +1,86 @@
+package org.coffeemine.app.spring;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.formlayout.FormLayout;
+
+
+class profileView extends VerticalLayout {
```

Comment:
Please name your class according to the file name. (and according to java standards e.g. has to start with capital letter) -> ProfileView.java as file name and ProfileView als class name.

Reply:
okay, will change to capital letter.